Resources/Prototypes/_Goobstation/Catalog/Fills/Briefcases/briefcase.yml

```
@@ -0,0 +1,21 @@
+  parent: BriefcaseSyndie
+  suffix: Syndicate, Dualetta Bundle
+  components:
+  - type: Item
+    size: Ginormous
+  - type: Storage
+    maxItemSize: Huge
+    grid:
+    - 0,0,6,3
+  - type: StorageFill
+    contents:
+      - id: WeaponPistolDualetta
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)